config/config.sample.php

```diff
@@ -166,70 +170,74 @@
  */
 
 /**
- * This sets the default language on your ownCloud server, using ISO_639-1
- * language codes such as `en` for English, `de` for German, and `fr` for
- * French. It overrides automatic language detection on public pages like login
- * or shared items. User's language preferences configured under "personal ->
- * language" override this setting after they have logged in.
+ * Define the default language of your ownCloud instance
```

Comment:
```suggestion
 * Defines the default language of your ownCloud instance
```

What convention are we using throughout this document?
e.g. up to here the first line has said `Defines something` `Indicates something`  rather than `Define something` `Indicate something`